infra/cmake/packages/AbseilConfig.cmake

```diff
@@ -36,4 +37,9 @@ function(_Abseil_import)
   set(Abseil_FOUND TRUE PARENT_SCOPE)
 endfunction(_Abseil_import)
 
+set(CMAKE_C_FLAGS_DEBUG     "${CMAKE_C_FLAGS_DEBUG} -fPIC")
+set(CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS_DEBUG} -fPIC")
+set(CMAKE_C_FLAGS_RELEASE   "${CMAKE_C_FLAGS_RELEASE} -fPIC")
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fPIC")
+
```

Comment:
@YiHyunJin Without this patch(-fPIC), absl does not work ? (It seems that this PR not closely relate to this PR.)

Reply:
Yes, if does not add this(-fPIC), it won't work...